pkg/kudoctl/cmd/version.go

```diff
@@ -30,5 +34,40 @@ func newVersionCmd() *cobra.Command {
 func VersionCmd(cmd *cobra.Command, args []string) error {
 	kudoVersion := version.Get()
 	fmt.Printf("KUDO Version: %s\n", fmt.Sprintf("%#v", kudoVersion))
+
+	// Print the Controller Version
+	controllerVersion, err := GetControllerVersion()
+	if err != nil {
+		fmt.Printf("KUDO Controller Version: %s\n", controllerVersion)
+	} else {
+		fmt.Printf("KUDO Controller Version: %#v\n", err)
+	}
+
 	return nil
 }
+
+// GetControllerVersion
+func GetControllerVersion() (string, error) {
+
+	controllerVersion := ""
+
+	client, err := kube.GetKubeClient(Settings.KubeConfig)
+	clog.V(3).Printf("Acquiring kudo client")
+	if err != nil {
+		clog.V(3).Printf("Failed to acquire kudo client")
+		return "", errors.New("<Failed to acquire kudo client>")
+	}
+
+	statefulsets, err := client.KubeClient.AppsV1().StatefulSets("").List(metav1.ListOptions{LabelSelector: "app=kudo-manager"})
```

Comment:
This needs to be tested because the KUDO manager may be in `kudo-system` or other namespaces. We need to ensure that this `List` can cover both cases.

Reply:
We should also use the existing function to pull the labelselectors:
e.g.
https://github.com/kudobuilder/kudo/blob/93b82209351abd3535629078bd24774e320e6a0a/pkg/kudoctl/kudoinit/setup/wait.go#L40